types/vote_set.go

```diff
@@ -560,6 +562,22 @@ func (voteSet *VoteSet) StringShort() string {
 		voteSet.height, voteSet.round, voteSet.signedMsgType, voteSet.maj23, frac, voteSet.votesBitArray, voteSet.peerMaj23s)
 }
 
+// LogString produces a logging suitable string representation of the
+// vote set.
+func (voteSet *VoteSet) LogString() string {
+	if voteSet == nil {
+		return nilVoteSetString
+	}
+	voteSet.mtx.Lock()
+	defer voteSet.mtx.Unlock()
+	voted, total, frac := voteSet.sumTotalFrac()
+
+	return fmt.Sprintf("Votes:%d/%d(%.3f) Validators:%q %v",
+		voted, total, frac,
+		voteSet.valSet,
```

Comment:
This might produce a large log when the validator set is large.

Reply:
True! I thought about that, but this has always been in the message content so we're not making it longer, and the prototype in the original data included some of this data as well, so I presumed that it was useful. I'm down to take it out, particularly if @ValarDragon thinks that would be useful.

We could also only put it in if the total number of votes was small or if the fraction was small, if that would be useful.

Reply:
I'm not really clear on what this is

Reply:
Same, same. Easy to get rid of it!